src/cephadm/CMakeLists.txt

```diff
@@ -2,3 +2,16 @@ if(WITH_TESTS)
   include(AddCephTest)
   add_tox_test(cephadm TOX_ENVS mypy)
 endif()
+
+add_custom_command(
+    OUTPUT "${CMAKE_SOURCE_DIR}/src/cephadm/cephadm"
```

Comment:
write to src/? why?

Reply:
The command we have to execute looks somewhat like so:

```
cd ceph/src
python3 -m zipapp cephadm  -p /usr/bin/python3
```
Unfortunately, this writes a file `cephadm.pyz` into ceph/src. Do you have a better idea?